nss-day-cohort-43/nuttyshell-the-coven-code
# Hh tasks

# Description
Created Tasks with the functionality to add, delete, and modify the 'complete' boolean.
Fixes # (issue)
## Type of change
# Testing Instructions
- Create a new task using the '+' button
- Cancel adding a new task using 'x'
- Delete a task using the 'x' on the individual tasks
- Click the checkbox and then go into your database to see the change from "complete": false to "complete": true
# Checklist:
